src/api/app/components/bs_request_activity_timeline.html.haml

```diff
@@ -0,0 +1,23 @@
+.row
+  .col-auto.pr-0
+    = image_tag('icons/activity-icon.svg', height: 18, title: 'Request notification')
```

Comment:
This icon can be replaced by one from Font Awesome: https://fontawesome.com/icons/wave-pulse?s=solid

Reply:
As discussed with @danidoni, I pushed changes for this.